uint16 counter
uint16 time
uint8 Motor_Servo_1
uint8 Motor_Servo_2
uint8 Motor_Servo_3
uint8 Motor_Servo_4
uint8 LED
uint8 status
